articles/node-vs-python/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: node-vs-python
+title: APIs in Node vs Python: A Comparison
+description: NodeJS and Python are among the most popular languages used to write APIs and backend services. Let's look at a comparison between the two most popular languages out there.
+author: saiharsha-balasubramaniam
+date: 2020-08-11T00:00:00-09:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/node-vs-python/hero.jpg
+    alt: node-vs-python
+---
+
+JavaScript and Python are the most popular languages among web developers. The idea of separating the front-end from the back-end as much as possible has birthed the idea of the JAMStack. JAMStack stands for **JavaScript, APIs and Markup**. It describes the idea of a website being completely static and client-side rendered. The website uses API calls to load all dynamic content. Let us look at the two most popular languages for writing APIs, Node and Python and compare and contrast the two.
+
+<!--more-->
+
+### Table of Contents
+
+- [The History](#the-history)
+- [Node vs Python: The Differences](#node-vs-python:-the-differences)
+- [APIs in Node](#apis-in-node)
+- [APIs in Python](#apis-in-python)
+- [Further Reading](#further-reading)
+
+### The History
+
+#### Node
+
+Node is a server-side javascript runtime. JavaScript was originally designed to run within a browser, inside a JavaScript engine like V8. To expand the capabilities of JavaScript outside the browser, Node was developed in **2009** by **Ryan Dahl**. Node combined the capabilites of the V8 JavaScript engine, an event loop and low level calls into a robust server-side scripting runtime. It soon grew into the preferred language by web developers.
```

Comment:
@cyberShaw  good morning Harsha  - just starting a preliminary review on your article - wanted to ask if this 1st sentence here is complete - Im not sure. 🤔 
Node is a server side JavaScript runtime ???

Reply:
Yes haha. Actually the definition of node was taken directly from the Node JS docs.

"Node is a Javascript Runtime based on the V8 Engine" 

So I think "Node is a Javascript Runtime" could also be written.